docs/sources/linking/data-link-variables.md

```diff
@@ -49,6 +49,17 @@ Value-specific variables are available under `__value` namespace:
 
 When linking to another dashboard that uses template variables, select variable values for whoever clicks the link.
 
-`${myvar:queryparams}` - where `myvar` is a name of the template variable that matches one in the current dashboard that you want to use.
-
-If you want to add all of the current dashboard's variables to the URL, then use `__all_variables`.
+- `${myvar:queryparams}` - where `myvar` is a name of the template variable that matches one in the current dashboard that you want to use the currently selected value of. 
```

Comment:
I'm not sure if my suggestion makes line 52 any clearer?

Reply:
I think it did, thank you. Sorry it took so long to incorporate, it has been a while since I looked at my change in the diff view so I was confused by the github diff display and was trying to figure out why you wanted to get rid of __all_variables and was trying to think of how to better demonstrate it when the rest of the PR does just that and also still mentions the _all_variables haha 😊